couchdbkit/version.py

```diff
@@ -5,5 +5,5 @@
 
 from __future__ import absolute_import
 from six.moves import map
-version_info = (0, 9, 0, 3, 8)
+version_info = (0, 9, 0, 4, 0)
```

Comment:
This is probably unnecesssarily long. maybe just 0.9.1 and it can be 1.0.0 once we support py3?

Reply:
Updated